webview/src/test/experimentsTable.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable unicorn/filename-case */
```

Comment:
In theory, it should already be in the ignore pattern of the rule (all files in test folders are ignored for PascalCase rule, but I think we had that because we used to have something like `util.tsx`. I'm torn between fixing the rule so that your file gets ignored and you can remove that line and just using PascalCase here (and even removing the ignore pattern on the test files).

Reply:
I forgot about this issue!
I prefer to ignore the rule for cases like this (no just tests).
My reason is that a PascalCase implies that the file is either a component (Like IconMenu.tsx) or a family of related components (like Row(s).tsx).
But for buildDynamicColumns.tsx and this file the jsx code is just ancillary stuff.
They are not the main purpose of the file.

Reply:
Then it should be in the ignore pattern of the rule. It is already the case, but it must be broken.